.github/workflows/test-autoreview.yml

```diff
@@ -18,6 +18,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
+permissions:
+  contents: read
+
```

Comment:
Do we not need the `pull_requests: read` or `write` for the `test-*.yml` workflows??

Reply:
No needed, you can see last PR on [https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/3797418235/jobs/6458336920](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/3797418235/jobs/6458336920)